docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_label: "Audit Scanner"
+title: "Kubewarden Audit Scanner"
+---
+
+# Kubewarden Audit Scanner
```

Comment:
suggestion: maybe we could call it "Background Audit Scanner"?
I would also remove "Kubewarden" from the title as it is a given I suppose.

Reply:
I think that "Background Audit Scanner" is too much. I would say that just "Audit Scanner" is good enough.